### ISSUE TITLE
Version 1.2.1 breaks build

### ISSUE THREAD

**Issue by Gachapen** [easy-webpack/config-sass#11]
Version 1.2.0 works fine, but when updating to 1.2.1, we can no longer build sass files. I don't see why this would happen based on the commits. From the warning message it seems like it doesn't use the sass loader. You can see an example warning below. Any ideas?
```
WARNING in ./src/about.sass
Module parse failed: [path censored]/src/about.sass Unexpected token (1:0)
You may need an appropriate loader to handle this file type.
| .pt-about-heading
| 	h2
| 		margin-bottom: 5px
 @ ./src ^\.\/.*$
 @ ./~/aurelia-loader-webpack/dist/commonjs/aurelia-loader-webpack.js
 @ multi aurelia
```

**Comment by niieani** [comment]
The dependent https://github.com/easy-webpack/config-css was updated to 3.0.0, which requires the latest version of Webpack. Sorry, I should have made a breaking change commit. Please stay at 1.2.0 if you're not planning on updating yet, there haven't been any changes, it's just cosmetic.

**Comment by Gachapen** [comment]
OK. Thanks for your answer. I will try to update webpack.

**Comment by niieani** [comment]
I'll try and republish so that more people don't encounter the same problem.

**Comment by Gachapen** [comment]
Thanks, that would be helpful.

**Comment by niieani** [comment]
Re-released as breaking change v2.0.0. 

v1.2.1 unpublished.

**Comment by larsfjerm** [comment]
@niieani Will this be released on https://www.npmjs.com/package/@easy-webpack/config-sass ?

**Comment by niieani** [comment]
It was published as 2.0.0. Not sure why the npm website isn't showing it. :/
